src/codeu/chat/client/ClientConversation.java

```diff
@@ -225,22 +234,38 @@ public int conversationsCount() {
   // If the input currentChanged is true, then re-establish the state of
   // the current Conversation, including its messages.
   public void updateAllConversations(boolean currentChanged) {
-	  System.out.println(this.getClass().toString() + " updateAllConversations()");
-    summariesByUuid.clear();
-    summariesSortedByTitle = new Store<>(String.CASE_INSENSITIVE_ORDER);
-    
-    summariesSortedByCreationTime = new TreeMap<ConversationSummary, String>(new sortByCreation());
-
-    for (final ConversationSummary cs : view.getAllConversations()) {
-      summariesByUuid.put(cs.id, cs);
-      summariesSortedByTitle.insert(cs.title, cs);
-      summariesSortedByCreationTime.put(cs, cs.title);
-    }
-
-    if (currentChanged) {
-      updateCurrentConversation();
-      messageContext.resetCurrent(true);
-    }
+	System.out.println(this.getClass().toString() + " updateAllConversations()");
```

Comment:
This goes along with all the other method calls that goes out to the command line.